.travis.yml

```diff
@@ -1,6 +1,5 @@
-language: generic
-
-sudo: false
+sudo: true
+language: haskell
```

Comment:
And why did you do this? It makes Travis install a GHC compiler, which we don't want (we do it manually).